// @bjerra/gatsby-theme-bd-salong/src/templates/services-page.js
import React from 'react'
import PropTypes from 'prop-types'
import Helmet from 'react-helmet'
import { graphql, Link } from 'gatsby'
import Layout from '../components/Layout'
import Content, { HTMLContent } from '../components/Content'

export const ServicesPageTemplate = ({
  title,
  helmet,
}) => {
  const PageContent = Content

  return (
    <section className="section">
      {helmet || ''}
      <div className="container content">
        <div className="columns">
          <div className="column is-10 is-offset-1">
            <h1 className="title is-size-2 has-text-weight-bold is-bold-light">
              {title}
            </h1>          
                
          </div>
        </div>
      </div>
    </section>
  )
}

ServicesPageTemplate.propTypes = {
  title: PropTypes.string,
  helmet: PropTypes.object,
}

const ServicesPage = ({ data }) => {
  const { dataJson: serviceList } = data
  const { title } = serviceList.categories[0]
  return (
    <Layout>
      <ServicesPageTemplate    
        helmet={
          <Helmet titleTemplate="%s | Blog">
            <title>{`${title}`}</title>
            <meta
              name="description"
            />
          </Helmet>
        }
        title={title}
      />
    </Layout>
  )
}

ServicesPage.propTypes = {
  data: PropTypes.shape({
    markdownRemark: PropTypes.object,
  }),
}

export default ServicesPage

export const pageQuery = graphql`
query serviceList {
  dataJson {
    id
    categories {
      title
    }
  }
}

`
